setup/metrics/format.go

```diff
@@ -6,7 +6,7 @@ const MB = 1 << 20
 
 // bytesToMBString converts the given number of bytes to Megabytes and returns it as a string formatted to 2 decimal places
 func bytesToMBString(bytes float64) string {
-	return fmt.Sprintf("%.2f", float64(bytes)/MB)
+	return fmt.Sprintf("%.2f MB", float64(bytes)/MB)
```

Comment:
I actually removed the `MB` in https://github.com/codeready-toolchain/toolchain-e2e/commit/12c3a6966a28f9e4bc876833b3b2f1b32687cd8e#diff-ecee3a957ef929c4d71d155152741b178f2566b5cc04cfb0a86ad93b97c50ec3

So let's just remove the `MB` from format_test.go instead.

Reply:
@rajivnathan the unit test fails with the following without MB
```   --- FAIL: TestBytesToMBString/success/zero (0.00s)
            format_test.go:18: 
                        Error Trace:    /Users/krana/go/src/github.com/codeready-toolchain/toolchain-e2e/setup/metrics/format_test.go:18
                        Error:          Not equal: 
                                        expected: "0.00"
                                        actual  : "0.00 MB"
                                        
                                        Diff:
                                        --- Expected
                                        +++ Actual
                                        @@ -1 +1 @@
                                        -0.00
                                        +0.00 MB
                        Test:           TestBytesToMBString/success/zero
```

Reply:
nvm, i am blind

Reply:
done in [ea2ffc7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1038/commits/ea2ffc75f4cc97ce2ad76ebf424647ad8ad67b91)